### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by BamBoozledMC** [eritislami/evobot#647] [Q&A]
Hey there, is there a dsicord server i can join to test this out before I download it?

**Comment by szvvx** [comment]
if you want to test the bot, just quickly create a test server

**Reply by BamBoozledMC** [reply]
Yea but i dont really want to jave to download the bot config it, make sure it works rhen create a discord server...

**Reply by wolf-yuan-6115** [reply]
> Yea but i dont really want to jave to download the bot config it, make sure it works rhen create a discord server...

yes it is working
